docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,258 @@
-:name: tab-risk-assessment
+:name: tab-ig-risk-assessment
 
 * - Statement
   - Guidance
   - Importance
 * - You must have a way to score risk to understand the underlying severity.
-  - You have a risk assesment methodology for scoring risks on multiple axes such as impact and likelihood.
+  - You have a risk assessment methodology for scoring risks on multiple axes such as impact and likelihood.
   - Mandatory
-* - You must have a process for mitigating risk using additional controls.
-  - Risks can be reduced to a level which brings it within agreed levels of appetite.
+* - You must carry out a data processing assessment for all projects requiring a TRE that are working with sensitive data.
+  - A data processing assessment is a process designed to identify risks arising out of the processing of sensitive data and to minimise these risks as far and as early as possible. This may take the form of an existing regulatory requirements such as Data Protection Impact Assessment.
   - Mandatory
-* - You must have an understanding of risk appetite.
-  - This includes understanding ownership of risk, and ability to accept risk which falls outside of the appetite should that become necessary.
+```
+
+**Risk Treatment:** The selection and implementation of strategies, controls, or measures to manage or mitigate identified risks, such as risk avoidance, risk transfer, risk reduction, or risk acceptance.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-risk-treatment
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a process for designing, implementing and recording risk mitigations where indicated by a risk assessment.
+  - Actions that are taken or not taken following a risk assessment must be recorded.
   - Mandatory
-* - You must carry out a data processing assessment for all projects requiring a TRE that are working with sensitive data.
-  - A data processing assessment is a process designed to identify risks arising out of the processing of sensitive data and to minimise these risks as far and as early as possible.
+```
+
+**Risk Ownership:** The assignment of responsibility and accountability to individuals or entities for managing and mitigating specific risks within the TRE organisation.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-risk-ownership
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a clear set of roles and responsibilities relating to risk including who owns risks and how they are escalated and delegated.
+  - The highest level of risk ownership is the top management of the TRE organisation. In order to ensure escalations to this level are rare suitable structures should be put in place to own, mitigate and accept risk.
+  - Mandatory
+* - You must understand risk appetite.
+  - This includes understanding ownership of risk, and ability to accept risk which falls outside of the appetite should that become necessary.
   - Mandatory
 ```
 
-## Project management
+## Study Management
```

Comment:
`Study Management` is a replacement for the `Project Management` section https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#project-management

Onboarding and Closure is copied, `Compliance checking` is new.